voting/src/voting.rs

```diff
@@ -14,16 +14,46 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-use crate::general_meeting::GeneralMeetingManager;
+use crate::common::Action;
+use crate::general_meeting::{GeneralMeeting, GeneralMeetingId, GeneralMeetingManager};
+pub use ckey::{Ed25519Public as Public, Signature};
 use coordinator::context::SubStorageAccess;
 use coordinator::module::*;
 use coordinator::types::*;
 use foundry_module_rt::UserModule;
 use parking_lot::RwLock;
+use primitives::H256;
+use rand::Rng;
 use remote_trait_object::raw_exchange::{import_service_from_handle, HandleToExchange, Skeleton};
 use remote_trait_object::{Context as RtoContext, Service, ServiceRef};
+use serde::{Deserialize, Serialize};
 use std::sync::Arc;
 
+enum ExecuteError {
+    InvalidMetadata,
+    InvalidSignature,
+    InvalidFormat,
+    GeneralMeetingNotFound,
+    VoteAfterEndTime,
+    VoteAfterTallyingTime,
+    InvalidAgendaNumber,
+    UsedVotePaper,
+}
+
+const CREATE_VOTE_PAPER_TX_TYPE: &str = "Vote_Paper";
```

Comment:
Let's use the constant value consistent.
```suggestion
const CREATE_VOTE_PAPER_TX_TYPE: &str = "Create_Vote_Paper";
```